docs/integrations/integration-cache.md

```diff
@@ -40,24 +40,24 @@ Note that this method overrides the existing object which is stored, so in order
 
 ### General usage
 ```python
-integration_context: Dict = demisto.getIntegrationContext()
+integration_context: Dict = demisto.get_integration_context()
```

Comment:
isn't it `integration_context: Dict = get_integration_context()` without the `demisto`

Reply:
good catch, will open a pr to fix this